custom/icds_reports/ucr/data_sources/ccs_record_cases_monthly_tableaunov17.json

```diff
@@ -3110,9 +3108,7 @@
         ]
       }
     },
-    "asynchronous": true,
```

Comment:
not putting this in the async queue. since we've made a ton of improvements and it's only one month I think this is doable in the celery task. will monitor it (also going to be useful to know to see how long rebuilds take now)

Reply:
Just realized that was a silly thing to say. It's definitely going in that queue.